src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1077,7 +1077,18 @@ emit_vector_create_elementwise (
 	MonoClass *vklass = mono_class_from_mono_type_internal (vtype);
 	MonoInst *ins = emit_xzero (cfg, vklass);
 	for (int i = 0; i < fsig->param_count; ++i) {
-		if (!is_zero_const (args [i])) {
+		if (is_zero_const (args [i])) {
```

Comment:
It seems to me that the same code got repeated multiple times. Can you create a helper function for it?